replit_river/codegen/server.py

```diff
@@ -155,11 +155,11 @@ def message_encoder(
     chunks = [
         f"def _{m.name}Encoder(",
         f"  e: {module_name}_pb2.{m.name}",
-        ") -> Dict[str, Any]:",
-        "  d: Dict[str, Any] = {}",
+        ") -> dict[str, Any]:",
+        "  d: dict[str, Any] = {}",
     ]
     # Non-oneof fields.
-    oneofs: DefaultDict[int, List[descriptor_pb2.FieldDescriptorProto]] = (
+    oneofs: DefaultDict[int, list[descriptor_pb2.FieldDescriptorProto]] = (
```

Comment:
What's the difference?

I wonder if it's like the stupid typescript thing where `String` (object) is different from `string` (literal/primitive) :grimacing:

Reply:
Python 3.8 and below did not permit the builtin types to be subscripted, `typing.List` and `typing.Dict` were added as future-compatible shims during the migration. Python 3.8 is EOL as of the end of Spooky Season, so now we can 🚀.